src/components/Searchbar.js

```diff
@@ -0,0 +1,20 @@
+import { BsSearch } from 'react-icons/bs';
+
+function Searchbar() {
+  return (
+
+    <div>
+
+      <input placeholder="search" required="" />
```

Comment:
- [ ] The search functionality does not filter through the data and thus, returns nothing. Please make sure the data can be filtered on this search.